packages/react-instantsearch-dom/src/components/__tests__/Stats.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Stats from '../Stats';
+
+describe('Stats', () => {
+  it('renders with default props', () => {
+    const tree = renderer.create(
+      <Stats nbHits={42} isSmartSorted={false} processingTimeMS={0} />
+    );
+
+    expect(tree.toJSON()).toMatchSnapshot();
+  });
+
+  it('accepts a custom className', () => {
+    const tree = renderer.create(
+      <Stats
+        isSmartSorted={false}
+        className="MyCustomStats"
+        nbHits={42}
+        processingTimeMS={0}
+      />
+    );
+
+    expect(tree.toJSON()).toMatchSnapshot();
+  });
+
+  it('renders rely on isSmartSorted', () => {
+    const tree = renderer.create(
+      <Stats
+        isSmartSorted={true}
+        nbHits={42}
+        nbSortedHits={21}
+        processingTimeMS={0}
+      />
+    );
+
+    expect(tree.toJSON()).toMatchSnapshot();
+  });
+
+  it('renders default implementation if nbHits is equal to nbSortedHits', () => {
+    const tree = renderer.create(
+      <Stats
+        isSmartSorted={true}
+        nbHits={42}
+        nbSortedHits={42}
+        processingTimeMS={0}
+      />
+    );
+
+    expect(tree.toJSON()).toMatchSnapshot();
```

Comment:
since these are new tests, what do you think about using inline snapshots? they seem small enough

Reply:
Sure, done https://github.com/algolia/react-instantsearch/pull/3010/commits/bf9509333d9a655e14411a46953c8ea8b1505b4b. Do you think it would make sense to do it for https://github.com/algolia/react-instantsearch/pull/3009 ?

Reply:
Yes, if not too big.